app/owner/OwnerService.py

```diff
@@ -0,0 +1,80 @@
+from flask import render_template, session, redirect, url_for, current_app, request
+from app import db
+from app.models import User, WorkSchedule, Role
+from app.owner import owner
+import datetime
+
+
+@owner.route('/ownerModule/ownerView')
```

Comment:
Macie jeden moduł zastanówcie się nad nazwami plików oraz nazwami endpointów. Nie używajcie dużych liter.
```suggestion
@owner.route('/owner/view')
```